redis/src/commands/implementer_macro.rs

```diff
@@ -0,0 +1,275 @@
+macro_rules! implement_commands {
```

Comment:
I wonder if we should just call this file `macros.rs`?

Reply:
sure, will fix